core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -2963,15 +3009,22 @@ class GroupCoordinatorTest {
     result
   }
 
-  private def leaveGroup(groupId: String, consumerId: String): LeaveGroupCallbackParams = {
-    val (responseFuture, responseCallback) = setupHeartbeatCallback
+  private def leaveGroup(groupId: String,
```

Comment:
Do we have any coordinator tests with multiple members? I just see the tests in `KafkaApis` which mock the coordinator.

Reply:
Good catch. I think we need to add some.